api/controllers/healthController.ts

```diff
@@ -11,7 +11,7 @@ export default class HealthController {
    */
   @GET
   @Path('/status')
-  async getStatus(): Promise<HealthDto> {
+  static async getStatus(): Promise<HealthDto> {
```

Comment:
These `static`s break the routing

Reply:
Can you write a test that tests this please?

Reply:
Probably! I'll make an attempt 👍